### ISSUE TITLE
Create documentation

### ISSUE THREAD

**Issue by aslagle** [NYPL-Simplified/opds-browser#28]

**Comment by EdwinGuzman** [comment]
I'll take this on since I'm trying to wrap my head around the codebase.

**Comment by EdwinGuzman** [comment]
Actually, there's some documentation already. Is this to add more details and explanations?

**Comment by aslagle** [comment]
Yeah it's been improved a lot since this issue was opened. See if you think there's anything that should still be added I guess.
